src/clarity/checkboxes/_checkbox.clarity.scss

```diff
@@ -0,0 +1,19 @@
+clr-checkbox {
+  label.indeterminate {
+
+    background-color: $clr-form-field-background-color;
+    border: none;
+
+    &:before {
+      width: 75px;
+      height: 75px;
+      border: 1px solid darkblue;
+    }
+
+    &:after {
+      width: 50px;
+      height: 50px;
+      border: 1px solid darkorange;
+    }
```

Comment:
Re: BEM. Great suggestion, @dragosrusu. For a number of reasons worthy of a blogpost, the team used Bootstrap as a base. So our classnames tend to reflect their direction. If/when we move away from Bootstrap, BEM, SMACSS, and OOCSS are all directions we should investigate.

re: standard sizes -- we usually try to do that (use standard sizes based on our vertical baseline). Kind of odd that there's a hardcoded 'darkorange' border there.

Thanks for the comments and suggestions!